src/component.ts

```diff
@@ -47,12 +48,14 @@ import {
   SamlStrategyFactoryProvider,
   SamlVerifyProvider,
 } from './strategies/SAML';
-import {AuthenticationConfig} from './types';
+import {AuthenticationConfig, IAuthSecureClientConfig} from './types';
 
 export class AuthenticationComponent implements Component {
   constructor(
     @inject(AuthenticationBindings.CONFIG, {optional: true})
     private readonly config?: AuthenticationConfig,
+    @inject(AuthenticationBindings.SecureClientConfig, {optional: true})
+    private readonly secureClientConfig?: IAuthSecureClientConfig,
```

Comment:
why new config ? Add it to existing AuthenticationConfig please

Reply:
changed